source/03-specifications/03-counter-report-common-attributes-and-elements.rst

```diff
@@ -115,48 +123,95 @@ Table 3.p (below): List of Data_Type Values
      - Reports
 
    * - Article
-     - An article, typically published in a journal or reference work.
+     - An article from a journal, or an article available as a standalone content item not aggregated into a title, for example in an institutional repository.
      - Repository\ |br|\ |lb|
        Scholarly_Collaboration_Network
-     - PR, IR\ |br|\ |lb|
-       PR_P1, IR_A1
+     - PR, IR
+
+   * - Audiovisual
+     - A form of multimedia, typically describing video content.
+     - Multimedia\ |br|\ |lb|
+       Multimedia_Collection
+     - PR, DR, IR
 
    * - Book
-     - A monograph text.
+     - A monograph text, edited volume, textbook, or other form of book that is not a reference work.
      - A&I_Database\ |br|\ |lb|
        Aggregated_Full_Content\ |br|\ |lb|
        Discovery_Service\ |br|\ |lb|
        eBook\ |br|\ |lb|
        eBook_Collection\ |br|\ |lb|
-       Repository\ |br|\ |lb|
-       Scholarly_Collaboration_Network
-     - PR, DR, TR, IR\ |br|\ |lb|
-       PR_P1, DR_D1, TR_B1, TR_B2, TR_B3
+     - PR, DR, TR
 
    * - Book_Segment
-     - A book segment (e.g. chapter, section, etc.).
+     - A segment of a book (e.g. chapter, section, etc.), or a segment available as a standalone content item not aggregated into a title, for example in an institutional repository.
+     
+       Where a whole book is being downloaded and there is no mechanism to determine the number of book segments, one Book_Segment should be reported in the IR.
      - Repository\ |br|\ |lb|
        Scholarly_Collaboration_Network
-     - PR, IR\ |br|\ |lb|
-       PR_P1
+     - PR, IR
 
-   * - Database
-     - A fixed database where content is searched and accessed in the context of the database. A given item on the host may be in multiple databases but a transaction must be attributed to a specific database. Note that Data_Type Database is only applicable for Searches and Access Denied at the database level and for Investigations and Requests for Full_Content_Databases*.
+   * - Conference
+     - A collection of papers, posters, or recordings of material associated with a conference. Typically part of a serial publication.
      - A&I_Database\ |br|\ |lb|
        Aggregated_Full_Content\ |br|\ |lb|
        Discovery_Service\ |br|\ |lb|
+       eBook\ |br|\ |lb|
        eBook_Collection\ |br|\ |lb|
-       Full_Content_Database\ |br|\ |lb|
-       Multimedia_Collection
-     - PR, DR\ |br|\ |lb|
-       PR_P1, DR_D1, DR_D2
+       eJournal\ |br|\ |lb|
+       Repository\ |br|\ |lb|
+       Scholarly_Collaboration_Network
```

Comment:
These Host_Types are only required to provide IR, so they shouldn't be listed here.